frontend/src/features/tasks/components/RepositorySelector.tsx

```
@@ -178,6 +184,41 @@ export default function RepositorySelector({
+        const data = await githubApis.getRepositories();
+        setRepos(data);
+        setCachedRepos(data);
+      }
+
+      toast({ title: t('branches.refresh_success') });
+    } catch {
+      toast({ variant: 'destructive', title: t('branches.refresh_failed') });
+    } finally {
+      setIsRefreshing(false);
+    }
+  }, [isRefreshing, currentSearchQuery, toast, t]);
```

Biome (2.1.2)

[error] This variable is used before its declaration.

The variable is declared here:


(lint/correctness/noInvalidUseBeforeDeclaration)